packages/typespec-client-generator-core/test/types.test.ts

```diff
@@ -2120,6 +2201,24 @@ describe("typespec-client-generator-core: types", () => {
       });
     });
 
+    it("multipart with encoding as binary bytes should not raise error", async function () {
```

Comment:
I don't think we should raise error if it's encoded as binary.

Reply:
I am not sure, "binary" is never an encoding we agreed on. It doesn't make much sense that you saying that bytes are encoded as bytes. it was just added to make the openapi3 output look good until we added support there.